### ISSUE TITLE
longterm values reset

### ISSUE THREAD

**Issue by JanDetrez** [hif2k1/battery_sim#50]
I see the long term values resetting for unknown reason.
* Might have been preceded by a Hassio restart
* Nothing is in my log files at this time stamp in "warning" mode

Can test if you want to dig more.

![resetvalue](https://user-images.githubusercontent.com/16916740/197499185-f8f30bf0-850e-479e-a3a7-00ab47635d82.JPG)

**Comment by hif2k1** [comment]
If it was associated with a restart it is possible it is due to the change I made to fix the energy dashboard problem. Energy dashboard seemed to be giving wrong readings around the time of restarts so I made the sensor values unavailable till all the values were restored, but if there were two resets in quick succession the "unavailable" value could replace the true value. Sorry. I will try and resolve this for the future.
